backend/src/main/java/com/google/sps/servlets/AddParticipantServlet.java

```diff
@@ -103,15 +120,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       // Match not found, insert participant entity into datastore
       Entity participantEntity = new Entity(KEY_PARTICIPANT);
       participantEntity.setProperty(PROPERTY_USERNAME, username);
-      participantEntity.setProperty(PROPERTY_STARTTIMEAVAILABLE, startTimeAvailable);
-      participantEntity.setProperty(PROPERTY_ENDTIMEAVAILABLE, endTimeAvailable);
+      participantEntity.setProperty(PROPERTY_STARTTIMEAVAILABLE, startTimeAvailable.toString());
+      participantEntity.setProperty(PROPERTY_ENDTIMEAVAILABLE, endTimeAvailable.toString());
       participantEntity.setProperty(PROPERTY_DURATION, duration);
       participantEntity.setProperty(PROPERTY_TIMESTAMP, timestamp);
       datastore.put(participantEntity);
     }
 
-    // Redirect back to the HTML page
-    response.sendRedirect("/index.html");
```

Comment:
no more redirection?

Reply:
For the prototype, we're redirecting back to the main page, but for the MVP, I think we're redirecting to the waiting page.

Reply:
The redirection will occur from the frontend instead